service/controller/key/key.go

```diff
@@ -16,27 +16,27 @@ const (
 )
 
 func AppOperatorAppName(getter LabelsGetter) string {
-	return fmt.Sprintf("%s-app-operator", ClusterID(getter))
+	return fmt.Sprintf("app-operator-%s", ClusterID(getter))
 }
 
 func AppOperatorValuesResourceName(getter LabelsGetter) string {
-	return fmt.Sprintf("%s-app-operator-values", ClusterID(getter))
+	return fmt.Sprintf("app-operator-values-%s", ClusterID(getter))
 }
 
 func BaseDomain(getter LabelsGetter, base string) string {
 	return fmt.Sprintf("%s.%s", ClusterID(getter), base)
 }
 
 func ChartOperatorAppName(getter LabelsGetter) string {
-	return fmt.Sprintf("%s-chart-operator", ClusterID(getter))
+	return fmt.Sprintf("chart-operator-%s", ClusterID(getter))
 }
 
 func ClusterValuesResourceName(getter LabelsGetter) string {
-	return fmt.Sprintf("%s-cluster-values", ClusterID(getter))
+	return fmt.Sprintf("cluster-values-%s", ClusterID(getter))
 }
 
 func ClusterCAName(getter LabelsGetter) string {
-	return fmt.Sprintf("%s-ca", ClusterID(getter))
+	return fmt.Sprintf("ca-%s", ClusterID(getter))
```

Comment:
Not sure if we should actually change the CA name. That will have significant impact if we change this for an existing cluster I think. But not sure either what the CA name should realistically be 😅